gdrapi.c

```diff
@@ -335,6 +344,90 @@ static void gdr_init_cpu_flags()
 
 // note: more than one implementation may be compiled in
 
+static void unroll8_memcpy(void *bar_ptr, const void *h_ptr, size_t size)
```

Comment:
Change the name "bar_ptr" and "h_ptr" because it is incorrect when this function is called from "gdr_copy_from_bar".